curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dce5f6ff907c2b2c47bb05.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 64dce5f6ff907c2b2c47bb05
+title: Step 21
+challengeType: 20
+dashedName: step-21
+---
+
+# --description--
+
+When `C` is empty, the disk should be moved necessarily from `A` to `C`. Add an `if` statement to check if `C` is empty. If it is, change `forward` to `True`.
```

Comment:
The prototype has `source, target, auxiliary` and I was simply keeping that order. But I agree with you, it would be better and more natural to change it.